src/plotting/create_plots.py

```diff
@@ -25,22 +25,28 @@
 log = logging.getLogger(__name__)
 
 
-def get_default_farmer_public_key() -> G1Element:
+def get_farmer_public_key(alt_fingerprint=None) -> G1Element:
     keychain: Keychain = Keychain()
-    sk_ent = keychain.get_first_private_key()
+    if alt_fingerprint is not None:
+        sk_ent = keychain.get_private_key_by_fingerprint(alt_fingerprint)
+    else:
+        sk_ent = keychain.get_first_private_key()
     if sk_ent is None:
         raise RuntimeError(
-            "No keys, please run 'chia keys generate' or provide a public key with -f"
+            "No keys, please run 'chia keys add', 'chia keys generate' or provide a public key with -f"
         )
     return master_sk_to_farmer_sk(sk_ent[0]).get_g1()
 
 
-def get_default_pool_public_key() -> G1Element:
+def get_pool_public_key(alt_fingerprint=None) -> G1Element:
```

Comment:
It's preferrable to specify python types for function arguments as well as local variables (although not the whole codebase has them yet).

def get_pool_public_key(alt_fingerprint: Optional[int] =None) -> G1Element:
sk_ent: Optional[Tuple[PrivateKey, bytes]] = .....

Reply:
Oh thanks for giving me feedback how to improve coding standards, will do this later today and update my PR.

Reply:
We've got a pile of PR's to merge today but yours will go in as soon as you make that update.

Reply:
I followed PEP 484 syntax for variable annotations. I just defined `sk_ent` type at the beginning of the function without initializing it since Tuples are immutable. It makes the code easier to read for the conditional branches. Thanks for helping me level up my Python skills @mariano54!